static/src/javascripts/projects/common/modules/crosswords/crossword.js

```diff
@@ -0,0 +1,720 @@
+define([
+    'react',
+    'bonzo',
+    'bean',
+    'fastdom',
+    'common/utils/$',
+    'common/utils/mediator',
+    'common/utils/detect',
+    'common/utils/scroller',
+    './anagram-helper/main',
+    './clues',
+    './controls',
+    './hidden-input',
+    './grid',
+    './helpers',
+    './keycodes',
+    './persistence',
+    'lodash/functions/debounce',
+    'lodash/collections/filter',
+    'lodash/collections/contains',
+    'lodash/collections/map',
+    'lodash/collections/some',
+    'lodash/arrays/zip',
+    'lodash/collections/forEach',
+    'lodash/collections/find'
+], function (
+    React,
+    bonzo,
+    bean,
+    fastdom,
+    $,
+    mediator,
+    detect,
+    scroller,
+    AnagramHelper,
+    Clues,
+    Controls,
+    HiddenInput,
+    Grid,
+    helpers,
+    keycodes,
+    persistence,
+    debounce,
+    filter,
+    contains,
+    map,
+    some,
+    zip,
+    forEach,
+    find
```

Comment:
I wonder how many of these lodash array methods we need any more? Certainly `filter`, `map`, `some` and `forEach` are part of ES5 and supported all the way back to IE9. We include an ES5 shim to support even older browsers.